packages/dashboard-backend/src/constants/schemas.ts

```diff
@@ -149,6 +149,46 @@ export const yamlResolverSchema: JSONSchema7 = {
   required: ['url'],
 };
 
+export const prebuiltResourcesSchema: JSONSchema7 = {
```

Comment:
Don't you mind renaming this const?

```suggestion
export const devWorkspaceResourcesSchema: JSONSchema7 = {
```

Reply:
I have fixed it